app/views/layouts/application.html.erb

```diff
@@ -49,6 +49,9 @@
               <ul class="dropdown-menu">
                 <li><%= link_to 'Logout', destroy_user_session_path, :method => :delete %></li>
                 <li><%= link_to "Apps", apps_path %></li>
+                <li><%= link_to "/users/#{current_user.id}/feedback" do %>
```

Comment:
You should use a named route here, rather than generating the path string manually. Create one by adding an `as` option to your route: `get 'users/:id/feedback', to: 'users#feedback', as: :user_feedback` gives you `user_feedback_path`.